ompi/mca/osc/rdma/osc_rdma.h

```diff
@@ -55,6 +55,8 @@
 
 #define RANK_ARRAY_COUNT(module) ((ompi_comm_size ((module)->comm) + (module)->node_count - 1) / (module)->node_count)
 
+#define MCA_OSC_RDMA_MAX_BTLS 2
```

Comment:
Maybe to make this constant more self-describing:
```suggestion
#define MCA_OSC_RDMA_MAX_USED_BTLS 2
```

Reply:
Done.